QModManager/HarmonyPatches/QuitToDesktop.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Original mod by RandyKnapp
+namespace QModManager.HarmonyPatches.QuitToDesktop
+{
+    using Harmony;
+    using UnityEngine;
+    using UnityEngine.UI;
+
+    internal static class QTD
```

Comment:
It looks like the `QTD` class and the `IngameMenu_Start_Patch` class could be merged

Reply:
`QTD` is also used in `IngameMenu_Open_Patch`